README.md

```diff
@@ -46,7 +46,7 @@ before the service runs. See the above "Configuration" section.
 
 ### Rubric Tester
 To run the rubric tester locally, create a python virtual environment at the top of the directory with:
-`python -m venv .venv`
+`python3 -m venv .venv`
```

Comment:
Maybe we should include instructions for python/ pyenv/ venv setup in the README

Reply:
great idea! Please take another look. I chose python 3.11 because that's what we use when we deploy aiproxy: https://github.com/code-dot-org/aiproxy/blob/3998faf2decf320dea89c8f83436618745e48732/Dockerfile#L1